deploy/helm/templates/events-collector-deployment.yaml

```diff
@@ -63,6 +63,29 @@ spec:
       imagePullSecrets:
         {{- toYaml .Values.imagePullSecrets | nindent 8 }}
       {{- end }}
+      {{- if and .Values.otel.swi_endpoint_check.enabled .Values.otel.metrics.swi_endpoint_check }}
```

Comment:
If the old property gets deprecated we don't want to check for both, we want to use the new value if it's set, and only use the old value if the new one isn't provided.